src/go/pt-k8s-pxc-recovery/recover/recover.go

```diff
@@ -0,0 +1,233 @@
+package recover
+
+import (
+	"errors"
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/percona/percona-toolkit/src/go/pt-k8s-pxc-recovery/helpers"
+)
+
+var clusterName = ""
+var clusterSize = 0
+var mostRecentPod int
+
+func SetClusterSize() error {
+	args := []string{
+		"get",
+		"pxc",
+		clusterName,
+		"-o",
+		"jsonpath='{.spec.pxc.size}'",
+	}
+	strSize, err := helpers.RunCmd(args...)
+	strSize = strings.Trim(strSize, "'")
+	if err != nil {
+		return err
+	}
```

Comment:
The error should be right checked after `RunCmd`

Reply:
done